test/box/fix-netbox-memory-corruption.result

```diff
@@ -0,0 +1,60 @@
+-- test-run result file version 2
```

Comment:
Look at the other netbox test files. They all have format `net.box_<description>.test.lua`. I propose to conform with the format.

Reply:
fix